9.2/root/usr/bin/run-postgresql

```diff
@@ -16,5 +16,10 @@ fi
 
 set_passwords
 
+if [ -f ${CONTAINER_SCRIPTS_PATH}/post-init ]; then
```

Comment:
Well, the original idea behind `post-init.sh` shell (I can even say Bash) script was that it would be by default an empty file with some comment only and this file would be really `sourced`, not run. The reason why sourcing might be more flexible here is that someone might want to use variables available defined at the point the script is being sourced.

On the other hand, this way is pretty complex and it would be fragile to break API of such run (e.g. by removing some expected variable definition). Also, `post-init` is now used in a place where not many usable variables are defined, so it makes more sense to make `post-init` a generic command -- by default make it just empty executable shell script that has a nice comment about what environment this script is run in.

Anyway, it is basically what is already proposed here now I believe. We just need to document that at the time of running `post-init`, the daemon is running locally with a socket file at `...` and even some working example how to load some SQL dump at this point should be provided.